src/utils/cardUtilityFunctions.js

```diff
@@ -354,3 +354,20 @@ export const chartValueFormatter = (value, size, unit, locale) => {
   }
   return `${renderValue}${!isNil(unit) ? ` ${unit}` : ''}`;
 };
+
+/**
+ * Charts render incorrectly if size is too small, so change their size to MEDIUM
+ * @param {string} size card size
+ */
+export const increaseSmallCardSize = (size, cardName) => {
+  const warnMsg = `${cardName} does not support card size ${size}`;
+  let returnSize;
+  if (size === CARD_SIZES.SMALL) {
+    console.warn(warnMsg); // eslint-disable-line no-console
```

Comment:
Should we have a test expecting a warning in the console when small or small-wide is selected?

Reply:
The warning should at the very least be limited to development environments via the `warning` package/fuction that is already installed as a dep.